get-resource.sh

```diff
@@ -1,59 +1,71 @@
-#!/bin/bash -xe
+#!/bin/bash
 #CACHEURL=http://172.22.0.1/images
 
+set -eux
+
 # Check and set http(s)_proxy. Required for cURL to use a proxy
-export http_proxy=${http_proxy:-$HTTP_PROXY}
-export https_proxy=${https_proxy:-$HTTPS_PROXY}
-export no_proxy=${no_proxy:-$NO_PROXY}
+export http_proxy="${http_proxy:-${HTTP_PROXY:-}}"
+export https_proxy="${https_proxy:-${HTTPS_PROXY:-}}"
+export no_proxy="${no_proxy:-${NO_PROXY:-}}"
+
+# configurable variables
+SHARED_DIR="${SHARED_DIR:-/shared}"
+CURL_VERBOSE="${CURL_VERBOSE:-}"
```

Comment:
One other things i have noticed also just suggestion.

I think instead of testing `[ -n "${CURL_VERBOSE}" ]` it would be nicer to have an addition ENABLE_VERBOSE_CURL variable that could be true or false,  then based on the result set the CURL_VERBOSE. I am just saying this because it is a bit weird when the user would want to turn verbose mode off,  the user would need to pass an empty string  to safely turn the verbose mode off, and that might throws off folks during a debugging session.

Reply:
Thanks. I changed the `CURL_VERBOSE` to be `true|false`, and in-script variable to `CURL_OUTPUT`.